### ISSUE TITLE
Team Editing Options

### ISSUE THREAD

**Issue by Applcore5** [GMDevinity/Flood#272]
Ive been thinking this for awhile and I'm aware certain aspects of this have been brought up before...but I'd like to bring them up as a whole.

•Allow Team Leader to make team public/non-public without having to destroy the team 
•Allow Team-Leader to change team-name (available only once a round and would make it easier to deal with offensive team names)
•Allow Team leader to change team color (available once a round)
•Allow Team Leader to transfer ownership (would help if leader plans on leaving) or Allow team leader to select "First Mate" ("First Mate" would retain ownership if current Team Leader leaves the team or leaves the game)

**Comment by Donkie** [comment]
Yes. I've been contemplating on things like this and also remaking the team menu. 

Good ideas!

**Comment by DawsonDevinity** [comment]
https://github.com/GMDevinity/Flood/issues/233

**Comment by Applcore5** [comment]
Dawson if possible you can combine them but both issues contain some individual suggestions that the others do not.

**Comment by DawsonDevinity** [comment]
Just edit your post to include mine.

**Comment by CounterCraftz** [comment]
I can possibly make a mockup if you want it @Donkie

**Comment by Donkie** [comment]
Sure if you want, I had some ideas but I can always take inspiration
